fc/parsing/CompactSyntaxParser.py

```diff
@@ -960,14 +648,7 @@ def _expr(self):
             return Simulations.Nested(args[2], args[0], args[1])
 
     class OneStepSimulation(BaseGroupAction):
-        def _xml(self):
-            attrs = {}
-            args = []
-            if 'step' in self.tokens:
-                attrs['step'] = str(self.tokens['step'][0])
-            if 'modifiers' in self.tokens:
-                args.append(self.tokens['modifiers'][0].xml())
-            return P.oneStep(*args, **attrs)
+        pass
```

Comment:
Again here we should document what the useful fields are. The one-step simulation class isn't (yet?) implemented in the Python version of FC.